dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -251,22 +251,23 @@ abstract class DLCWallet
     */
   override def cancelDLC(dlcId: Sha256Digest): Future[Unit] = {
     for {
-      inputs <- dlcInputsDAO.findByDLCId(dlcId)
-      dbs <- spendingInfoDAO.findByOutPoints(inputs.map(_.outPoint))
-      // allow this to fail in the case they have already been unreserved
-      _ <- unmarkUTXOsAsReserved(dbs).recover { case _: Throwable => () }
-
       dlcOpt <- dlcDAO.read(dlcId)
-      _ = dlcOpt match {
+      dlcDb = dlcOpt match {
         case Some(db) =>
           require(
             db.state == DLCState.Offered || db.state == DLCState.Accepted || db.state == DLCState.Signed,
```

Comment:
Is `Signed` a safe state here? Doesn't this imply you have sent your signatures to your counterparty?

Reply:
We have #3162 for this